index.html

```diff
@@ -1,19 +1,114 @@
+  <head>
+    <meta charset="UTF-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, shrink-to-fit=no"
+    />
     <title>Karma</title>
-    <link href='https://fonts.googleapis.com/css?family=Roboto:400,500,300' rel='stylesheet' type='text/css'>
-    <link rel="stylesheet" href="css/normalize.css">
-    <link rel="stylesheet" href="css/style.css">
-    <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
-</head>
-<body>
-
+    <link
+      href="https://fonts.googleapis.com/css?family=Roboto:400,500,300"
+      rel="stylesheet"
+      type="text/css"
+    />
+    <link rel="stylesheet" href="css/normalize.css" />
+    <link rel="stylesheet" href="css/style.css" />
+    <link rel="shortcut icon" type="image/x-icon" href="favicon.ico" />
+  </head>
+  <body>
     <!-- Add your HTML markup here -->
     <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
     <!-- All the images you need are in the 'img' folder -->
+    <header>
+      <div class="logo-container">
+        <img src="img/karma-logo.svg" alt="karma logo" class="karma" />
+      </div>
+      <nav class="navbar">
+        <ul class="navbar-nav">
+          <li class="nav-item">
+            <a class="nav-link" href="/html/">Meet Karma</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="//">How it works</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="//">Store</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="//">Blog</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="//">Help</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="/">Login</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <div class="first-background">
+        <img
+          src="img/first-background.jpg"
+          alt="a person with a laptop"
+          class="main-img"
+        />
+
+        <h1>Introducing Karma</h1>
+        <p>Bring WiFi with you,everywhere you go .</p>
+        <button class="btn">Learn More</button>
+      </div>
+    </main>
+    <section class="services-container">
+      <h2 class="services-header">Everyone needs a little Karma.</h2>
+      <div class="icons-container">
+        <div class="icons">
+          <img src="img/icon-devices.svg" alt="desktop-icon" class="icon1" />
+          <p>Internet for all device.</p>
+        </div>
+        <div class="icons">
+          <img src="img/icon-coffee.svg" alt="coffe-icon" class="icon1" />
+          <p>Boost your producrivity.</p>
+        </div>
+        <div class="icons">
+          <img src="img/icon-refill.svg" alt="refill-icon" class="icon1" />
+          <p>Pay as You Go.</p>
+        </div>
+      </div>
+    </section>
 
-</body>
```

Comment:
i think this one is <body>

Reply:
<body>

Reply:
Thank you for your review Anu , i will check it out and implement the changes needed